index.bs

```diff
@@ -837,11 +837,11 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 					1. Abort these steps.
 
-				1. Let |data| be a copy of the [=system clipboard data=] represented as [=clipboard items=]. For the MIME types defined in the [=mandatory data types=] list, |data| contains the sanitized copy of text/plain format.
+				1. Let |data| be a copy of the [=system clipboard data=] represented as [=clipboard items=]. For the MIME types defined in the [=mandatory data types=] list, |data| contains the content of text/plain format.
 
 					Issue: Some OSs contain multiple clipboard (e.g. Linux, "primary", "secondary", "selection"). Define from which of those data is read.
 
-					Issue: Add definition of sanitized copy.
+					Issue: Currently Safari does some sanitization. See https://github.com/w3c/clipboard-apis/issues/150.
```

Comment:
Not sure if we need this issue if we have agreement on saying data MAY be sanitized above.

Reply:
Removed.